### PR TITLE
De sx hsc

### PR DESCRIPTION
**Pull request by damonge**

### CODE REVIEW

File: ClLike/cl_like/cl_like.py [hunk 1]
```diff
@@ -141,9 +138,14 @@ def _init_globals(self):
...
             # Halo model correction for the transition regime
             if self.HM_correction == 'halofit':
+                from .hm_extra import HalomodCorrection
                 self.hmcorr = HalomodCorrection()
             else:
                 self.hmcorr = None
+        elif self.bias_model == 'LagrangianPT':
+            from .lpt import LPTCalculator, get_lpt_pk2d
+        elif self.bias_model == 'EulerianPT':
+            from .ept import EPTCalculator, get_ept_pk2d
```
Comment (damonge) [inline ClLike/cl_like/cl_like.py:148]: will these be accessible to other methods of the class?

File: ClLike/cl_like/cl_like.py [hunk 2]
```diff
@@ -322,19 +324,38 @@ def _get_nz(self, cosmo, name, **pars):
...
         if (self.nz_model == 'NzShiftWidth') or (self.nz_model == 'NzWidth'):
             wz = pars.get(self.input_params_prefix + '_' + name + '_wz', 1.)
-        z = zm+dz+(z-zm)/wz
-        msk = z >= 0
-        z = z[msk]
-        nz = nz[msk]
-        return (z, nz)
+            jacob = wz
+        # NzShiftParam parametrized as z_true - z_false = f(z_true)
+        if (self.nz_model == 'NzShiftParamExp'):
```
Comment (damonge) [inline ClLike/cl_like/cl_like.py:339]: do you think we need all of them? Just for simplicity, should we keep whatever you used for the actual paper? Up to you!

File: ClLike/cl_like/cl_like.py (hunk 1 above)
Comment (carlosggarcia) [reply ClLike/cl_like/cl_like.py:148]: No. I have just realized we discussed exactly this in the `imports` branch. I thought they were independent. Let me fix this

File: ClLike/cl_like/cl_like.py (hunk 2 above)
Comment (carlosggarcia) [reply ClLike/cl_like/cl_like.py]: I've removed the Exp case that we did not use in the end.

File: ClLike/cl_like/cl_like.py (hunk 1 above)
Comment (carlosggarcia) [reply ClLike/cl_like/cl_like.py]: Fixed following https://github.com/xC-ell/xCell-likelihoods/pull/5/files#r729558435

**Review by damonge — COMMENTED** [review]
For some reason it won't let me approve (it says I'm an author of this PR)...

**Comment by damonge** [comment]
so I just went ahead and merged

**Pull request merged**